src/renderer/components/main/middlebar/toolbar_search.tsx

```diff
@@ -37,9 +54,10 @@ class Search extends React.Component<any, undefined> {
 
 export default connect ({
   container: Main,
-  shouldComponentUpdate: false,
+  shouldComponentUpdate: true,
```

Comment:
I'm thinking this isn't entirely correct but I wasn't sure what the correct value should be.